src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -693,74 +691,74 @@ private void testFeedbackResponseCommentActions() throws Exception {
-        resultsPage.verifyContainsElement(By.id("frComment-visibility-options-trigger-0-1-0-1-1"));
-        resultsPage.verifyCommentRowContent("-0-1-0-1-2", "test comment 2", "Teammates Test");
-        resultsPage.verifyContainsElement(By.id("visibility-options-0-1-0-1-2"));
+        resultsPage.addFeedbackResponseComment("showResponseCommentAddForm-1-0-1-1", "test comment 1");
+        resultsPage.addFeedbackResponseComment("showResponseCommentAddForm-1-0-1-1", "test comment 2");
+        resultsPage.verifyCommentRowContent("-1-1-0-1-2", "test comment 1", "Teammates Test");
+        resultsPage.verifyContainsElement(By.id("frComment-visibility-options-trigger-1-1-0-1-2"));
+        resultsPage.verifyCommentRowContent("-1-1-0-1-3", "test comment 2", "Teammates Test");
+        resultsPage.verifyContainsElement(By.id("visibility-options-1-1-0-1-3"));
 
         resultsPage.verifyHtmlMainContent("/instructorFeedbackResultsAddComment.html");
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr", "Open Session");
         resultsPage.displayByRecipientGiverQuestion();
         resultsPage.loadResultSectionPanel(0, 1);
-        resultsPage.verifyCommentRowContent("-0-0-1-1-1", "test comment 1", "Teammates Test");
-        resultsPage.verifyCommentRowContent("-0-0-1-1-2", "test comment 2", "Teammates Test");
+        resultsPage.verifyCommentRowContent("-1-0-1-1-2", "test comment 1", "Teammates Test");
+        resultsPage.verifyCommentRowContent("-1-0-1-1-3", "test comment 2", "Teammates Test");
 
         resultsPage.loadResultSectionPanel(1, 2);
-        resultsPage.addFeedbackResponseComment("showResponseCommentAddForm-1-1-1-1", "test comment 3");
-        resultsPage.verifyCommentRowContent("-1-1-1-1-1", "test comment 3", "Teammates Test");
+        resultsPage.addFeedbackResponseComment("showResponseCommentAddForm-2-0-1-1", "test comment 3");
+        resultsPage.verifyCommentRowContent("-2-1-0-1-1", "test comment 3", "Teammates Test");
 
         ______TS("Typical case: edit existing feedback response comment");
 
-        resultsPage.editFeedbackResponseComment("-1-1-1-1-1", "edited test comment");
-        resultsPage.verifyCommentRowContent("-1-1-1-1-1", "edited test comment", "Teammates Test");
+        resultsPage.editFeedbackResponseComment("-2-1-0-1-1", "edited test comment");
+        resultsPage.verifyCommentRowContent("-2-1-0-1-1", "edited test comment", "Teammates Test");
         resultsPage.verifyHtmlMainContent("/instructorFeedbackResultsEditComment.html");
 
         ______TS("Typical case: edit comment created by different instructor");
 
         resultsPage.editFeedbackResponseComment("-1-0-1-1-1", "Comment edited by different instructor");
-        resultsPage.verifyCommentRowContent("-1-1-1-1-1", "edited test comment", "Teammates Test");
+        resultsPage.verifyCommentRowContent("-1-0-1-1-1", "Comment edited by different instructor", "Teammates Test");
```

Comment:
Just curious, why is the comment also changing? Shouldn't only index of the id change?

Reply:
@tshradheya It corresponds to the above `editFeedbackResponseComment` comment edit. The original indexing does not seem to match the line above directly as well.